staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -6557,8 +6557,8 @@ type LocalObjectReference struct {
 	// Name of the referent.
 	// More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
 	// TODO: Add other useful fields. apiVersion, kind, uid?
-	// +optional
-	Name string `json:"name,omitempty" protobuf:"bytes,1,opt,name=name"`
+	// +required
+	Name string `json:"name" protobuf:"bytes,1,opt,name=name"`
```

Comment:
probably need to take "opt" out of the tags?

Reply:
Hmm, many others keep it.  Maybe not?  I forget the rules on this.

@alexzielenski @jpbetz

Reply:
definitely keep opt in the proto tags ... we don't want to break proto parsing